ledger/sandbox/src/main/scala/com/digitalasset/platform/sandboxnext/Runner.scala

```diff
@@ -121,6 +123,7 @@ class Runner(config: SandboxConfig) extends ResourceOwner[Port] {
                   config.metricsReporter,
                   config.metricsReportingInterval,
                 )
+                lfValueTranslationCache: LfValueTranslation.Cache = Cache.none
```

Comment:
Why not come up with a decent default? For the state value cache, we cap it at 128 MB in Sandbox.

Reply:
I think 0 is a decent default, considering that it's the default with the other set of options and that its effect is really minor for the average sandbox user. I'll add this to the options in a way that is consistent across sandbox-classic and sandbox.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6052/commits/84e260534bcdf942034701834c03f59440b2b134